diffsims/generators/rotation_list_generators.py

```diff
@@ -58,6 +58,44 @@ def get_fundemental_zone_grid(space_group_number, resolution):
     fz_grid_axangle = reduce_to_fundemental_zone(raw_grid_axangle, zone_string)
     return _returnable_eulers_from_axangle(fz_grid_axangle,'rzxz',round_to=2)
 
+def get_grid_streographic(crystal_system,resolution,equal='angle'):
+    """
+    Creates a rotation list by combining the minimum region of the streogram's beam directions
+    with in plane rotations
+
+    Parameters
+    ----------
+    crytal_system : string
+        'cubic','hexagonal','tetragonal','orthorhombic','monoclinic','trigonal' add 'triclinc' which acts as 'none'
+
+    resolution : float (angle in degrees)
+        Nearest neighbour rotations are seperated by a distance of 'resolution'
+
+    equal : 'angle' or 'area'
+        See docstrings for diffsims.utils.gridding_utils.get_beam_directions
+        
+    Returns
+    -------
+    rotation_list : 
+    """
+    from itertools import product
+    beam_directions_rzxz = beam_directions_to_euler_angles(get_beam_directions(crystal_system,resolution,equal=equal))
+    beam_directions_szxz = beam_directions_rzxz.to_AxAngle().to_Euler(axis_convention='szxz') # convert to high speed convention
```

Comment:
Do we risk any loss of precision with all these conversions? I doubt it, but just thought I'd check.

Reply:
https://github.com/pyxem/diffsims/blob/746efbc23a9f4210b498cdc2d923b60f26264bcd/diffsims/tests/test_utils/test_rotation_conversion_utils.py#L82

Suggests we can go from euler to axangle and axangle to euler within `1e-8` so I'm not very concerned about this.